.gitignore

```diff
@@ -132,7 +133,7 @@ TAGS
 *.tar.gz
 
 # packaging related files
-!packaging/scripts/prepare_deb_freerdp-nightly.sh
```

Comment:
please remove this change, changes to these scripts require changes in the build infrastructure.